app/pages/login/login.js

```diff
@@ -152,7 +156,7 @@ export let Login = translate()(React.createClass({
   renderBrowserWarning: function() {
     const { t } = this.props;
 
-    if (!utils.isChrome()) {
+    if (!utils.isAcceptedBrowser()) {
```

Comment:
Why changing from Chrome to accepted browsers (FF, Edge), tests are only made only on Chrome by Tidepool and we know that some glitches can appear on FF and edge... the reason why we display this message for all non chrome users.
```suggestion
    if (!utils.isChrome()) {
```